source/docs/assist/using_voice_assistants_overview.markdown

```diff
@@ -0,0 +1,19 @@
+---
+title: "Using voice assistants - overview"
+---
+
+We are in [chapter two](https://www.home-assistant.io/blog/2023/04/27/year-of-the-voice-chapter-2/) of Home Assistant's Year of the Voice.
```

Comment:
This should have been a relative link to the root of the website. 

- `https://www.home-assistant.io/blog/2023/04/27/year-of-the-voice-chapter-2/` -> `/blog/2023/04/27/year-of-the-voice-chapter-2/`